public/base-locales/de.json

```diff
@@ -603,5 +603,24 @@
   "on_block_msg": "Du wurdest aufgrund deiner Mitgliedschaft blockiert",
   "denied": "verweigert",
   "showcase": "Showcase",
-  "unknown_event": "unbekanntes Event"
-}
\ No newline at end of file
+  "unknown_event": "unbekanntes Event",
+  "scanner_countdown": "verfügbar in {{time}}s",
+  "success": "Erfolg",
+  "contribute": "Beitragen",
+  "mi": "mi",
+  "filter_by_areas": "nach Gebieten filtern",
+  "submit_nest_name": "Nestnamen mitteilen",
+  "nest_submission_menu": "Namen aktualisieren",
+  "nest_submissions": "Nest Einreichungen",
+  "nest_submissions_subtitle": "Ermöglicht neue Namen für Nester einzureichen.",
+  "pokemon_opacity": "dynamische Pokemon Durchsichtigkeit",
+  "invasion_opacity": "dynamische Invasion Durchsichtigkeit",
+  "raid_opacity": "dynamische Raid Durchsichtigkeit",
+  "opacity_ten_minutes": "Durchsichtigkeit ab 10 Minuten",
```

Comment:
Opacity is more like "Deckkraft",right? :)

Reply:
yeah in the context it's better to use Durchsichtigkeit, I agree on that one :) Discussed with author :)